compute/metadata/go.mod

```diff
@@ -0,0 +1,5 @@
+module cloud.google.com/go/compute/metadata
+
+go 1.19
+
+require cloud.google.com/go/compute v1.12.0
```

Comment:
is the intent to drop this in a subsequent version of compute/metadata?

Reply:
Yes, this is just required to make the transition some and for the modules to resolve. This process of carving out an existing module is documented on the go wiki if your curious: https://github.com/golang/go/wiki/Modules#is-it-possible-to-add-a-module-to-a-multi-module-repository